opencti-platform/opencti-graphql/src/database/utils.js

```diff
@@ -326,6 +326,19 @@ export const extractIdsFromStoreObject = (instance) => {
   return ids;
 };
 
+export const extractObjectsRestrictionsFromInputs = (inputs) => {
```

Comment:
This method should be a bit more controlled on the type of the element.
objects are linked to container.
You also only extract the markings and not the other parts of ACL (orga sharing + authorized members)
Constant should also be available instead of using 'objects' and 'object-marking'

Reply:
You're right for the sharing and the authorized members.
For the input, its more about testing if the entity type is a container in order to get the related restrictions instead of just rely on the inputs